### DISCUSSION TITLE
Práctica 2 de HTML

### DISCUSSION THREAD

**Discussion by Manuel-F-e** [Launch-X-Latam/MisionFrontEnd#454] [Team Posts]
Hola! Alguien sabe dónde puedo encontrar la práctica del módulo de HTML? Ya que, solo veo que están los temas pero no las indicaciones de la práctica. Gracias

**Comment by mafer23** [comment]
en la parte final de cada modulo estan las practicas.

**Comment by Manuel-F-e** [comment]
> en la parte final de cada modulo estan las practicas.

Claro, pero en el de HTML no viene así. Por eso pregunto.

**Comment by nathalieCarrasco** [comment]
hola tengo acceso a gibhub y mi cuenta de temas pero aun no me asignan
equipo para ver las clases soy de front
ayuda 

https://teams.microsoft.com/l/channel/19:WJNKiwxGQtPIrd14ayE6FvxZqYOEO2lBT092fLCDGk01@thread.tacv2/General?groupId=8c234d62-8e08-415a-8242-509fa69c5c40&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816

**Comment by MrAlizarinCrimson** [comment]
Yo tampoco tengo equipo.. ya estoy en teams y github

**Comment by Josephr1318** [comment]
***@***.***/General?groupId=8c234d62-8e08-415a-8242-509fa69c5c40&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816

**Comment by RichardPB96** [comment]
lo de los equipos no es al final junto a un proyecto?
